internal/handlers/common/aggregations/operators/accumulators/sum.go

```diff
@@ -54,6 +53,7 @@ func newSum(accumulation *types.Document) (Accumulator, error) {
 	case int32, int64:
 		accumulator.number = expr
 	default:
+		// TODO if the value is a document, evaluate it as operators.Operator
```

Comment:
Do we have an issue number for this TODO?

Reply:
Fixed